traits/util/tests/test_trait_documenter.py

```diff
@@ -118,29 +125,19 @@ def test_get_definition_tokens(self):
 
     def test_add_line(self):
 
-        src = textwrap.dedent(
-            """\
-        class Fake(HasTraits):
-
-            #: Test attribute
-            test = Property(Bool, label="ミスあり")
-        """
-        )
         mocked_directive = mock.MagicMock()
 
         documenter = TraitDocumenter(mocked_directive, "test", "   ")
-        documenter.object_name = "Property"
+        documenter.object_name = "test_attribute"
+        documenter.parent = Fake
 
         with mock.patch(
-            "traits.util.trait_documenter.inspect.getsource", return_value=src
+            (
+                "traits.util.trait_documenter.ClassLevelDocumenter"
+                ".add_directive_header"
+            )
```

Comment:
nit : this would be more readable if you don't mind ignoring the line length limit.

Reply:
I don't think I want to change the formatting in this PR: these lines are only unindented with the removal of the outer `with mock.patch` block. I'd prefer to leave this as it is.